plugins/tiddlywiki/geospatial/tests/widgets/geomap-empty-layer.tid

```diff
@@ -0,0 +1,12 @@
+title: $:/plugins/tiddlywiki/geospatial/tests/widgets/geomap-empty-layer
```

Comment:
I'm guessing this test can't currently be expressed as a compound tiddler? Is it just the requirement to include the geospatial plugin that is the problem?

Reply:
Yes. I don't see a way to include the plugin using compound tiddler. Without the plugin, it crashes with infinite bounds exception.

I think that is the same reason you need to disable the other geomap widget tests (the note in those testcase tiddlers says it fails on CI with playwright, but it will fail in any browser situation).

Reply:
Hi @btheado I think we should fix that issue. It would take a new "import" field on compound tiddlers.

Reply:
(it would be similar to #8447)

Reply:
Sure, I'll make a pull request for that.

Reply:
I made the pull request at #8499.

I changed from testcase widget to testcase tiddler. It requires the aforementioned #8499 to be merged, otherwise it gives infinite bounds exception when viewed in the browser.